pallets/eth-bridge/src/lib.rs

```diff
@@ -977,11 +995,25 @@ impl<T: Config> Pallet<T> {
             .into_iter()
             .filter(|v| v.account_id == account_id)
             .nth(0)
-            .unwrap();
+            .ok_or_else(|| ())?;
 
         submit_ethereum_events::<T>(validator, events_partition, signature)
     }
 
+    pub fn submit_latest_ethereum_block_vote(
```

Comment:
what is the difference between this and `submit_latest_ethereum_block()`?

Reply:
one is exposed from the pallet, one is part of the call module.
submit_latest_ethereum_block is following the conversion of the pallet for calls.

I can flatten submit_latest_ethereum_block into the vote if you prefer that. I need it though as part of the pallet interface so It can be called via the runtime api